clang/lib/DPCT/APINames.inc

```diff
@@ -355,17 +355,17 @@ ENTRY(cudaProfilerStart, cudaProfilerStart, false, NO_FLAG, P0, "comment")
 ENTRY(cudaProfilerStop, cudaProfilerStop, false, NO_FLAG, P0, "comment")
 
 // Atomic API
-ENTRY(atomicAdd, atomicAdd, true, NO_FLAG, P0, "Successful")
-ENTRY(atomicSub, atomicSub, true, NO_FLAG, P0, "Successful")
-ENTRY(atomicExch, atomicExch, true, NO_FLAG, P0, "Successful")
-ENTRY(atomicMin, atomicMin, true, NO_FLAG, P0, "Successful")
-ENTRY(atomicMax, atomicMax, true, NO_FLAG, P0, "Successful")
-ENTRY(atomicInc, atomicInc, true, NO_FLAG, P0, "Successful")
-ENTRY(atomicDec, atomicDec, true, NO_FLAG, P0, "Successful")
-ENTRY(atomicCAS, atomicCAS, true, NO_FLAG, P0, "Successful")
-ENTRY(atomicAnd, atomicAnd, true, NO_FLAG, P0, "Successful")
-ENTRY(atomicOr, atomicOr, true, NO_FLAG, P0, "Successful")
-ENTRY(atomicXor, atomicXor, true, NO_FLAG, P0, "Successful")
+ENTRY(atomicAdd, atomicAdd, true, NO_FLAG, P0, "Successful: DPCT1039")
+ENTRY(atomicSub, atomicSub, true, NO_FLAG, P0, "Successful: DPCT1039")
+ENTRY(atomicExch, atomicExch, true, NO_FLAG, P0, "Successful: DPCT1039")
+ENTRY(atomicMin, atomicMin, true, NO_FLAG, P0, "Successful: DPCT1039")
+ENTRY(atomicMax, atomicMax, true, NO_FLAG, P0, "Successful: DPCT1039")
+ENTRY(atomicInc, atomicInc, true, NO_FLAG, P0, "Successful: DPCT1039")
+ENTRY(atomicDec, atomicDec, true, NO_FLAG, P0, "Successful: DPCT1039")
```

Comment:
in #595 , Diagnostic information is set to successful, without DPCT1039.  Please double check whether you need to add DPCT1039

Reply:
Yes, use `--no-use-generic-space=true` to migrate this code:
```
__device__ void addByte(unsigned int *s_WarpHist, unsigned int data) {
  atomicDec(s_WarpHist + data, 1);
}

```